### ISSUE TITLE
unable to query for the element using bounding boxes generated using IBM accessibility checker npm package

### ISSUE THREAD

**Issue by swathitella** [IBMa/equal-access#693]
I am using accessibility checker npm package to generate the bounding box coordinates for a webpage.

When I run this command, **npx achecker --outputFormat json --outputFolder c:/achecker --reportLevels pass https://accessibility.18f.gov/keyboard/#**, I get the results that were attached.

When I use the generated bounding box result top and left values to get that element from DOM using document.elementFromPoint(x, y), I am not able to retrieve the desired element.
[accessibility.18f.gov#.zip](https://github.com/IBMa/equal-access/files/8075433/accessibility.18f.gov.zip)

Any insights?

**Comment by tombrunet** [comment]
Element positions are going to be dependent on the page width at the time of the scan. If you do not have the exactly same page and page width, you might not get the same element. Since you're using the headless command-line tool, I'm not sure how you'd ensure that you are using the same page width when you revisit the page. You should use the xpath to find the element.

The bounding box was intended for a situation in which you took a screenshot with the check using the automated tools, so that the screenshot and captured positions should align. Does this answer the question?

**Comment by aliunwala** [comment]
@swathitella Did Tom's response help?
